consensus/polybft/statesyncrelayer/state_sync_relayer.go

```diff
@@ -136,7 +144,11 @@ func (r *StateSyncRelayer) AddLog(log *ethgo.Log) {
 
 			if err := r.executeStateSync(stateSyncProof); err != nil {
 				r.logger.Error("Failed to execute state sync", "err", err)
+
+				continue
 			}
+
+			r.logger.Info("State sync executed", "stateSyncID", i)
```

Comment:
Should we lower this log level to debug log?

Reply:
This can be useful to have it in info, not too much details